admin/coverage

```diff
@@ -4,18 +4,25 @@ set -e
 
 MAIN_CRATE=portable-rustls
 
+ALL_FEATURES_EXCEPT_UNSAFE=$(admin/all-features-except unsafe-assume-single-core $MAIN_CRATE)
+
+ALL_FEATURES_EXCEPT_UNSAFE_AND_BROTLI=$(admin/all-features-except unsafe-assume-single-core,brotli $MAIN_CRATE)
+
+# (also need to exclude `unstable-api-all` which includes `zlib` in first case)
+ALL_FEATURES_EXCEPT_UNSAFE_AND_ZLIB=$(admin/all-features-except unsafe-assume-single-core,unstable-api-all,zlib $MAIN_CRATE)
```

Comment:
nit: need to exclude `unstable-api-all` from both of these to try testing these separately with almost all other features below

```suggestion
# (also need to exclude `unstable-api-all` which includes both `brotli` & `zlib`)
ALL_FEATURES_EXCEPT_UNSAFE_AND_BROTLI=$(admin/all-features-except unsafe-assume-single-core,brotli $MAIN_CRATE)
ALL_FEATURES_EXCEPT_UNSAFE_AND_ZLIB=$(admin/all-features-except unsafe-assume-single-core,unstable-api-all,zlib $MAIN_CRATE)
```